timescaledb/how-to-guides/multinode-timescaledb/multinode-ha.md

```diff
@@ -0,0 +1,27 @@
+# High availability (HA) with multinode
+Efficiently myocardinate multidisciplinary meta-services and just in time action
```

Comment:
We already have this section on "native replication": https://docs.timescale.com/timescaledb/latest/how-to-guides/distributed-hypertables/enable-native-replication/

But I think it makes sense to also talk more generally about HA here, including different options and alternatives.